3696/3/app/views/home.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card-columns">
+  <% Place.find_each do |place| %>
```

Comment:
all queries and data preparation should be suited in controller actions and passed into views